compiler/plugins/input/Torch/InputConversion/ConvertTMTensorToLinalgExt.cpp

```diff
@@ -84,10 +84,17 @@ getStandardAttentionIndexingMaps(MLIRContext *ctx) {
       AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {k2, k1}, ctx);
   AffineMap vMap =
       AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {k2, n}, ctx);
+  AffineMap sMap = AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0,
+                                  SmallVector<AffineExpr>{}, ctx);
```

Comment:
nit: this can be simpler:

```suggestion
  AffineMap sMap = AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, ctx);
```

https://github.com/llvm/llvm-project/blob/403897484f939cffd9b813eb0b759d7113f5295b/mlir/include/mlir/IR/AffineMap.h#L56-L59

side note [no action required]: we can use `auto` for all of these because `XXX::get` already spells the type. See https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable

Reply:
Ok noted, thanks